src/Dmr.Api/Controllers/DMRController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Dmr.Api.Models;
+using Dmr.Api.Services.AsyncProcessor;
+using Microsoft.AspNetCore.Mvc;
+using System.Text;
+
+namespace Dmr.Api.Controllers
+{
+    [Route("/messages")]
+    [ApiController]
+    public class DmrController : ControllerBase
+    {
+        private readonly IAsyncProcessorService<Message> processor;
+
+        public DmrController(IAsyncProcessorService<Message> processor)
+        {
+            this.processor = processor;
+        }
+
+        /// <summary>
+        /// Accepts 'Post'ed messages with Buerokratt headers and an encoded/encrypted message body. 
+        /// </summary>
+        /// <param name="headers">Model bound headers</param>
+        /// <returns>204 Accepted or 400 BadRequest</returns>
```

Comment:
Looks like specific return types, this might be appropriate:
```suggestion
        /// <returns>204 Accepted or 400 BadRequest</returns>
        [ProducesResponseType(StatusCodes.Status204NoContent)]
        [ProducesResponseType(StatusCodes.Status400BadRequest)]
```